File: Easy/FindTheIndexOfTheFirstOccurrenceInAString/SolutionTest.java
package Easy.FindTheIndexOfTheFirstOccurrenceInAString;
import org.junit.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class SolutionTest{
    
    @Test
    public void strStrTest(){
        assertEquals(0, Solution.strStr("sadbutsad", "sad"));
    }

    @Test
    public void strStrTest2(){
        assertEquals(-1, Solution.strStr("leetcode", "leeto"));
    }
}

File: Easy/FindTheIndexOfTheFirstOccurrenceInAString/SolutionTest.py
from unittest import TestCase, main
from Solution import Solution

class SolutionTest(TestCase):
    def test_strStrTest(self):
        self.assertEquals(0, Solution.strStr("sadbutsad", "sad"))


    def test_strStrTest2(self):
        self.assertEquals(-1, Solution.strStr("leetcode", "leeto"))


if __name__ == "__main__":
    main()
